partner_label_report/readme/USAGE.rst

```diff
@@ -0,0 +1 @@
+Print Partner Label report
```

Comment:
Something like this:

Go to Contacts / Partners
Select the partners you want to print
Click on Print > Label report


(adapt to the correct menu names ,etc)..